src/people.rs

```diff
@@ -291,10 +488,26 @@ impl ContextPeopleExt for Context {
 #[cfg(test)]
 mod test {
     use super::{ContextPeopleExt, PersonCreatedEvent, PersonId, PersonPropertyChangeEvent};
-    use crate::{context::Context, people::PeoplePlugin};
-    use std::{cell::RefCell, rc::Rc};
+    use crate::{
+        context::Context,
+        people::{PeoplePlugin, PersonPropertyHolder},
```

Comment:
Why do you need PersonPropertyHolder.

Reply:
Because I am calling `non_derived_dependencies()` in the test